tests/CSharpTemp/CSharpTemp.h

```diff
@@ -509,3 +509,27 @@ class DLL_API TestOverrideFromSecondaryBase : public Foo, public SecondaryBase
     void VirtualMember();
     void setProperty(int value);
 };
+
+class DLL_API TestParamToInterfacePassBaseOne
+{
+};
+
+class DLL_API TestParamToInterfacePassBaseTwo
+{
+	int m;
```

Comment:
Done :)
I assumed zero as default value for integers. But, I remember now that it's only for global variables. :P